internal/store/node.go

```diff
@@ -57,6 +58,29 @@ func nodeMetricFamilies(allowAnnotationsList, allowLabelsList []string) []genera
 	}
 }
 
+func createNodeDeletionTimestampFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGeneratorWithStability(
+		"kube_node_deletion_timestamp",
```

Comment:
Do we want to set this metric as Alpha/Experimental first, and then promote this into Stable?

Because, labels/metric name can't be changed if it's a stable metric.

Reply:
Good catch! @rexagod can you introduce it as experimental?

Reply:
ACK, fixed.